dbt/adapters/presto/connections.py

```diff
@@ -172,21 +148,24 @@ def open(cls, connection):
             return connection
 
         credentials = connection.credentials
-        if credentials.method == 'kerberos':
-            auth = KerberosAuthentication()
+        if credentials.method in ['ldap', 'kerberos']:
+            auth = prestodb.auth.BasicAuthentication(credentials.user, credentials.password)
+            http_scheme = "https"
         else:
             auth = prestodb.constants.DEFAULT_AUTH
+            http_scheme = None
```

Comment:
I believe the default value is `'http'` (`prestodb.constants.HTTP`). `None` will do unkind things.

Reply:
Fixed: https://github.com/fishtown-analytics/dbt-presto/pull/10/commits/4ce347d3128958241f0ce2d767a041aaa563eee9